tests/robot-tests/tests/libs/common.robot

```diff
@@ -771,3 +771,15 @@ lookup or return webelement
 user closes Set Page View box
     user clicks element    id:pageViewToggleButton
     user waits until element is not visible    id:editingMode
+
+user checks page for details section
+    [Arguments]    ${heading}
+    user checks page contains element    testid:Expand Details Section ${heading}
```

Comment:
Doesn't feel like we need this as we have `user checks page for details dropdown`